swdb2017/brain_observatory/utilities/pearson_corr_coeff.py

```diff
@@ -0,0 +1,32 @@
+import numpy as np
+
+def pearson_corr_coeff(x, y):
+
+    '''
+    Arguments:
```

Comment:
should be Parameters

Reply:
also: https://docs.scipy.org/doc/scipy-0.14.0/reference/generated/scipy.stats.pearsonr.html